README.md

```diff
@@ -1,62 +1,12 @@
-  - [ ] **TODO**: Remove issue labels that do not add much value. Suggest removing `duplicate`, `good first issue`, `help wanted`, `invalid`, `question`.
-  - [ ] **TODO**: Add issue labels as needed. Suggest adding `triage`.
-- Github CI:
-  - [ ] **TODO**: Under `.github/workflows`, update the provided examples as appropriate for your toolchain. Suggest starting with linting and tests. Suggest having each check as a separate workflow, so if there is a failure, its easy to tell at a glance what it was.
+## Contributing
+We welcome contributions! Please see our [contributing guidelines](https://github.com/hubmapconsortium/user-templates-api/blob/development/CONTRIBUTING.md). For adding new templates, please see our [template contributing guidelines](https://github.com/hubmapconsortium/user-templates-api/blob/development/src/user_templates_api/templates/jupyter_lab/templates/new_template/README.md). 
 
-<hr>
+We also have a slack channel for feedback on the workspaces. Please reach out to the HuBMAP Help Desk ([help@hubmapconsortium.org](mailto:help@hubmapconsortium.org)) to get in touch.
 
-# **TODO**: your-repo-name-here
 
-**TODO**: Short description; Also fill in the repo description in Github so there's something beyond just the repo name in the top-level list of repos.
-
-**TODO**: Describe how this relates to the rest of HuBMAP, and in particular how it relates to the Portal. Is this a workflow that prepares data for the portal? Is it an API that will be used by the portal? Is this an ancillary web site the portal should link out to? Please make sure the Portal maintainers are aware of the integration plans.
-
-**TODO**: Diagram: Is there any part of the architecture or process that is easier to understand as a picture? Options:
-
-```mermaid
-graph LR
-    mermaid
-    mermaid --> md[This is in the markdown!]
-    mermaid --> link[Links are supported!]
-    click link href "https://mermaid-js.github.io/mermaid-live-editor/"
-    mermaid --> limit[Diagram types are constrained!]
-```
-
-[![Google Drawing](https://docs.google.com/drawings/d/e/2PACX-1vQgMhQbVUk_PLGo8chvxONDchqtf7OwvKS0QUiihv4JhDy4-IuF0m5GZWl3nMwLB7CuJ5feDbgQPh73/pub?w=350)](https://docs.google.com/drawings/d/1rMHDXndMnuXXtqf6s387_Pa-T0UfgZCnenFOvQ5dsdQ/edit)
-
-![draw.io](example-for-hubmap-template-repo.drawio.svg)
-
-**TODO**: Think about the audiences for your repo and structure the documentation accordingly, proceeding from outsiders to insiders:
-- Users: Where is this deployed? Is there separate API documentation? How should bugs be reported?
-- New developers: How do I set up a dev environment? How do I prepare a PR?
-- Maintainers: How are releases and deployments managed?
\ No newline at end of file
+## Contributors
+This project is part of the HuBMAP consortium. The main contributors to the workspaces are the [ittsburgh Supercomputing Cente](https://www.psc.edu/) and the [HIDIVE Lab](https://hidivelab.org) at [Harvard Medical School](https://hms.harvard.edu).
```

Comment:
```suggestion
This project is part of the HuBMAP consortium. The main contributors to the workspaces are the [Pittsburgh Supercomputing Center](https://www.psc.edu/) and the [HIDIVE Lab](https://hidivelab.org) at [Harvard Medical School](https://hms.harvard.edu).
```

Reply:
Thanks for updating the README! 
Suggestions for fixing typos.

Reply:
Thanks for seeing that! I copied it over from the site, but clearly didn't copy it well.